pkg/opt/config.go

```diff
@@ -118,12 +142,16 @@ func NewConfig() (*Config, error) {
 	viper.SetEnvPrefix("relay_pls")
 	viper.AutomaticEnv()
 
+	viper.SetDefault("metrics_enabled", false)
 	viper.SetDefault("oauth_vault_engine_mount_root", DefaultOAuthVaultEngineMountRoot)
 	viper.SetDefault("vault_engine_mount", DefaultVaultEngineMount)
 
 	config := &Config{
 		Debug: viper.GetBool("debug"),
 
+		MetricsEnabled: viper.GetBool("metrics_enabled"),
+		MetricsAddr:    viper.GetString("metrics_server_addr"),
```

Comment:
We may as well standardized on a default value for this. `localhost:3050` is what we've been using to now, and I'm fine with that port if you are.

Reply:
That works.